packages/cpp/ArmoniK.Api.Worker/header/Worker/ProcessStatus.h

```diff
@@ -14,8 +14,8 @@ class ProcessStatus {
   [[nodiscard]] bool ok() const { return ok_; }
   [[nodiscard]] const std::string &details() const { return details_; }
 
-  static const ProcessStatus OK;
-  static const ProcessStatus ERROR;
+  static const ProcessStatus PROCESS_OK;
+  static const ProcessStatus PROCESS_ERROR;
```

Comment:
I am not sure what is the purpose of this change. Those constants are namespaced by the class itself.

Reply:
Apparently it conflicts with some macros on Windows (though it shouldn't be an issue since it's the worker, thus Linux only)

Reply:
Yes on Windows for MSVS there is a conflict with the ERROR macro. Although, the conflict was before we added namespaces.

Reply:
Macros are not namespaced, and thus will still be in conflict with the globals. That being said, I'm not a fan of SCREAMING_SNAKE_CASE here.
```suggestion
  static const ProcessStatus Ok;
  static const ProcessStatus Error;
```